docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -11,6 +12,8 @@ This directory provides ways to restrict who can access the functions of a contr
 
 - {Ownable} is a simple mechanism with a single "owner" role that can be assigned to a single account.
 This mechanism can be useful in simple scenarios, but fine grained access needs are likely to outgrow it.
+- {OwnableTwoStep} is like Ownable, but ownership transfer is a two-step process where the newly proposed
```

Comment:
This is not a different component, so it shouldn't be presented as a separated entity. The OwnableComponent Embeddable Implementations section should be updated accordingly (adding the new impls), and we can maybe add a note below stating that we should use one of the implementations depending on if we want two-steps transfers or not.